### ISSUE TITLE
Investigate EchoNest API integration

### ISSUE THREAD

**Issue by toroidal-code** [spectra-music/spectra#10]
Utilizing a music info system like [Echonest](http://developer.echonest.com/docs/v4/index.html) will allow us to improve the overall user experience.

Things like detailed Genre, Album, and Artist information (like google music), better song recommendations, and dynamic playlists based on the user's library.

Possible questions
- Where would we integrate this? The Rails side, and then provide the data to the client? or the client-side?

**Comment by toroidal-code** [comment]
I'm leaning towards the rails side. We could have a `create_dynamic_playlist_with(model => item)` function in the `PlaylistsController` for example

**Comment by nickserv** [comment]
I agree about doing this on the backend. It seems like it would be too closely tied to the models and backend logic for it to be done efficiently on the fronted.

**Comment by kristenmills** [comment]
I server side is better for that for performance reasons.

**Comment by zellio** [comment]
+1 server side - if it proves really time expensive shell it off to job threads

**Comment by toroidal-code** [comment]
possible gem: [echonest-ruby-api](https://github.com/maxehmookau/echonest-ruby-api)
